internal/services/network/application_gateway_resource.go

```diff
@@ -250,24 +251,6 @@ func resourceApplicationGateway() *pluginsdk.Resource {
 							ValidateFunc: validation.IntBetween(1, 86400),
 						},
 
-						"authentication_certificate": {
-							Type:     pluginsdk.TypeList,
-							Optional: true,
-							Elem: &pluginsdk.Resource{
-								Schema: map[string]*pluginsdk.Schema{
-									"name": {
-										Type:     pluginsdk.TypeString,
-										Required: true,
-									},
-
-									"id": {
-										Type:     pluginsdk.TypeString,
-										Computed: true,
-									},
-								},
-							},
-						},
-
```

Comment:
We cannot remove this argument. See this doc about the migration https://learn.microsoft.com/en-us/azure/application-gateway/retirement-faq, existing V1 application gateway is still supported until April 2026.